### ISSUE TITLE
Generate password is not working.

### ISSUE THREAD

**Issue by zsemo** [nilsteampassnet/TeamPass#2399]
### Steps to reproduce
1. Add Item
2. Clickk Password & Visibility
3. Click Generate icon

### Expected behaviour
Generate password is work fine.

### Actual behaviour
Add new item, then click Password & Visibility and click Generate password icon, appear empty Information popup window. Click Close and nothing happens.

### Server configuration
**Operating system**: Windows NT 10.0 build 14393 (Windows Server 2016) AMD64

**Web server:** Apache/2.4.27 (Win64) PHP/7.1.9

**Database:** 5.7.19

**PHP version:** 7.1.9

**Teampass version:** 2.1.27.17

**Teampass configuration file:**
```
'max_latest_items' => '10',
'enable_favourites' => '1',
'show_last_items' => '1',
'enable_pf_feature' => '0',
'log_connections' => '1',
'log_accessed' => '1',
'time_format' => 'H:i:s',
'date_format' => 'd/m/Y',
'duplicate_folder' => '0',
'item_duplicate_in_same_folder' => '0',
'duplicate_item' => '1',
'number_of_used_pw' => '3',
'manager_edit' => '1',
'cpassman_dir' => 'C:/wamp/www/TeamPass',
'cpassman_url' => 'http://<anonym_url>/TeamPass
'favicon' => 'http://<anonym_url>/TeamPass/favicon.ico',
'path_to_upload_folder' => 'C:/wamp/www/TeamPass/upload',
'url_to_upload_folder' => 'http://<anonym_url>/TeamPass/upload',
'path_to_files_folder' => 'C:/wamp/www/TeamPass/files',
'url_to_files_folder' => 'http://<anonym_url>/TeamPass/files',
'activate_expiration' => '0',
'pw_life_duration' => '0',
'maintenance_mode' => '0',
'enable_sts' => '0',
'encryptClientServer' => '1',
'cpassman_version' => '2.1.27',
'ldap_mode' => '1',
'ldap_type' => '',
'ldap_suffix' => '',
'ldap_domain_dn' => '',
'ldap_domain_controler' => '',
'ldap_user_attribute' => '0',
'ldap_ssl' => '0',
'ldap_tls' => '0',
'ldap_elusers' => '0',
'ldap_search_base' => '0',
'richtext' => '0',
'allow_print' => '1',
'roles_allowed_to_print' => '0',
'show_description' => '1',
'anyone_can_modify' => '0',
'anyone_can_modify_bydefault' => '0',
'nb_bad_authentication' => '10',
'utf8_enabled' => '1',
'restricted_to' => '0',
'restricted_to_roles' => '0',
'enable_send_email_on_user_login' => '0',
'enable_user_can_create_folders' => '1',
'insert_manual_entry_item_history' => '0',
'enable_kb' => '1',
'enable_email_notification_on_item_shown' => '0',
'enable_email_notification_on_user_pw_change' => '1',
'custom_logo' => '',
'custom_login_text' => '',
'default_language' => 'english',
'send_stats' => '0',
'get_tp_info' => '1',
'send_mail_on_user_login' => '0',
'nb_items_by_query' => 'auto',
'enable_delete_after_consultation' => '0',
'enable_personal_saltkey_cookie' => '0',
'personal_saltkey_cookie_duration' => '31',
'email_smtp_server' => '',
'email_smtp_auth' => 'false',
'email_auth_username' => '',
'email_auth_pwd' => '<removed>'
'email_port' => '25',
'email_security' => 'none',
'email_server_url' => 'http://<anonym_url>/TeamPass',
'email_from' => '',
'email_from_name' => 'TeamPass',
'pwd_maximum_length' => '40',
'google_authentication' => '0',
'delay_item_edition' => '0',
'allow_import' => '1',
'proxy_ip' => '',
'proxy_port' => '',
'upload_maxfilesize' => '50mb',
'upload_docext' => 'doc,docx,dotx,xls,xlsx,xltx,rtf,csv,txt,pdf,ppt,pptx,pot,dotx,xltx',
'upload_imagesext' => 'jpg,jpeg,gif,png',
'upload_pkgext' => '7z,rar,tar,zip',
'upload_otherext' => 'sql,xml',
'upload_imageresize_options' => '1',
'upload_imageresize_width' => '800',
'upload_imageresize_height' => '600',
'upload_imageresize_quality' => '90',
'use_md5_password_as_salt' => '0',
'ga_website_name' => '',
'api' => '0',
'subfolder_rights_as_parent' => '1',
'show_only_accessible_folders' => '0',
'enable_suggestion' => '0',
'otv_expiration_period' => '7',
'default_session_expiration_time' => '60',
'duo' => '0',
'enable_server_password_change' => '0',
'ldap_object_class' => '0',
'bck_script_path' => '',
'bck_script_filename' => 'bck_cpassman',
'timezone' => '',
'menu_type' => 'context',
'syslog_enable' => '0',
'syslog_host' => 'localhost',
'syslog_port' => '514',
'ldap_object_class' => '0',
'saltkey_ante_2127' => 'vsZpF6EJbB6cQVT9',
'teampass_version' => '2.1.27',
'migration_to_2127' => 'done',
'manager_move_item' => '0',
'create_item_without_password' => '0',
'send_stats_time' => '1531259703',
'agses_authentication_enabled' => '0',
'personal_saltkey_security_level' => '0',
'item_extra_fields' => '0',
'ldap_new_user_is_administrated_by' => '0',
'ldap_port' => '389',
'offline_key_level' => '0',
'enable_http_request_login' => '0',
'otv_is_enabled' => '0',
'ldap_and_local_authentication' => '0',
'secure_display_image' => '1',
'upload_zero_byte_file' => '0',
'upload_all_extensions_file' => '0',
'files_with_defuse' => 'done',
'send_statistics_items' => 'stat_country;',
'ldap_allowed_usergroup' => '',
'tree_counters' => '0',
'disable_show_forgot_pwd_link' => '1',
'can_create_root_folder' => '0',

```

**Updated from an older Teampass or fresh install:**

### Client configuration

**Browser:** Chrome - 68.0.3440.106

**Operating system:** Windows - 64bits

#### Web server error log
```
Constant COST already defined - C:\wamp\www\TeamPass\includes\config\settings.php (19)
```

#### Teampass 10 last system errors
```
 * 02/08/2016 12:11:57 - Query: INSERT INTO `teampass_kb_items` (`kb_id`,`item_id`) VALUES (2, '')<br />Error: Incorrect integer value: '' for column 'item_id' at row 1<br />@ 
 * 02/08/2016 12:11:50 - Query: INSERT INTO `teampass_kb_items` (`kb_id`,`item_id`) VALUES (1, '')<br />Error: Incorrect integer value: '' for column 'item_id' at row 1<br />@ 
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```
![image](https://user-images.githubusercontent.com/42453667/44786041-2854fa00-ab93-11e8-8744-9460637bafe9.png)

**Comment by nilsteampassnet** [comment]
This was fixed in 2.1.27.17 which seems to be your version too.

Currently I cannot reproduce.

Perhaps should you clean your cache? or try using an inconito window

**Comment by zsemo** [comment]
I cleaned cache, tried another browser and incongnito mode, but unfortunately the same error.
